server/src/main/java/io/druid/segment/realtime/FireDepartmentMetrics.java

```diff
@@ -103,6 +108,18 @@ public void incrementHandOffCount(){
     handOffCount.incrementAndGet();
   }
 
+  public void setSinkCount(long sinkCount){
+    this.sinkCount.set(sinkCount);
+  }
+
+  public void reportMessageMaxTimestamp(long messageMaxTimestamp){
+    this.messageMaxTimestamp.set(Math.max(messageMaxTimestamp, this.messageMaxTimestamp.get()));
+  }
+
+  public void reportSourceMaxTimestamp(long sourceMaxTimestamp){
```

Comment:
It is reserved for tasks who can provide source timestamp(kafka 0.10.0, https://issues.apache.org/jira/browse/KAFKA-2511), I wonder should I document it in metrics.md?